web/src/locales/fr.json

```
@@ -121,11 +120,7 @@
     "now": "Maintenant",
     "by-source": {
       "emissions": "Émissions carbone",
-      "electricity-production": "Production d'électricité",
-      "electricity-consumption": "Consommation d'électricité",
-      "total-emissions": "Émissions totales de carbone",
-      "total-electricity-production": "Production totale d'électricité",
-      "total-electricity-consumption": "Consommation totale d'électricité"
+      "total-emissions": "Émissions totales de carbone"
```

```suggestion
      "electricity-mix": "mix électrique",
      "total-emissions": "Émissions totales de carbone"
```

Seems like the most used phrasing for "electricity mix" as far as I can tell